Preparation/MyStack/MyStackInter.cs

```diff
@@ -9,6 +9,9 @@ public class MyStack<T> : IEnumerable<T>
         private int _count = -1;
         private T[] _underliyngArray = new T[5];
 
+        public event Action<T, bool> ItemAdd;
+        public event Action<T, bool> ItemDelete;
```

Comment:
Do you actually need a boolean parameter now?)

Reply:
And maybe change the events naming. As we are in the context of Stack class, these should be, respectively, ItemPushed and ItemPopped.